recipe/meta.yaml

```
@@ -26,21 +26,41 @@ requirements:
     - numpy >=1.23.5
     - scipy >=1.8.0
     - pandas >=2.2.0
-    - python-tzdata # [win]
+    - python-tzdata  # [win]
+    - tzdata  # [win]
```

Does it work only with python-tzdata?
```suggestion
    - tzdata  # [win]
```

---

Looks like it works. Thanks